support-modules/rest/src/main/scala/com/gu/monitoring/SafeLogger.scala

```diff
@@ -1,21 +1,33 @@
 package com.gu.monitoring
```

Comment:
this is a second copy of SafeLogger - it was probably from when the mono repo was created.  There's no obvious module at the top of the dependency tree to put a single copy in so I've just left the duplication